README.md

```diff
@@ -1,2 +1,40 @@
-# memo
-Memo is an Anki mobile app written in Flutter
+# Memo
+
+Memo is an Anki mobile app written in Flutter. This repository holds the source code for the hybrid application - currently iOS & Android versions - of Memo.
```

Comment:
I think this is overly verbose, as the latter quote mentions that this targets iOS and Android only.

suggestion to simply put:
"Memo is a programming-oriented Anki hybrid application, written in Flutter"